testing/ncids-js-testing/public/example-pages/promoblock-example.html

```diff
@@ -210,13 +210,13 @@ <h2 class="nci-promo-block__heading">
 								researchers from diverse populations in cancer and cancer health
 								disparities research.
 							</p>
-							<a
+							<button
 								href="https://www.cancer.gov"
 								alt="More Ongoing Research Studies"
 								class="usa-button usa-button--secondary"
 							>
 								Learn More
```

Comment:
let's get some screen reader text behind this button for accessibility

Reply:
oh that's right, paul k did this in his too so one of us will get conflicts too :)